pastis/config_pastis.ini

```diff
@@ -126,10 +130,11 @@ tel_size_px = 1024
 ;px_size_nm = 18e3  ; in detector plane
 im_size_px_webbpsf = 320
 im_size_px_pastis = 1024
-
+im_size_lamD_hcipy = 30
+z_pup_downsample = 10
 ; this is not used automatically in the functions, it is always defined (or read from here) manually
-current_analysis = 2020-01-13T21-34-29_luvoir-small
-
+current_analysis = 2021-02-23T15-35-34_luvoir-small
+sampling = 4.
```

Comment:
Like above, this indicates old usage of the sampling parameter (which simply changed in the meantime).
Now, each instrument (LUVOIR, HiCAT, ...) has their separate sampling parameter, so please go wherever you still call ('numerical', 'sampling') and change it accordingly.

Reply:
@ananyasahoo1904 here

Reply:
ok